docs/Buttons/GetCapabilities/index.md

```diff
@@ -10,13 +10,20 @@ Purpose
 : Get the capabilities of buttons in the vehicle.
 
 ### Request
+SDL requests HMI to provide capabilities of supported buttons and saves received capabilities internally.
 
 #### Parameters
 
 This RPC has no additional parameter requirements
 
 ### Response
 
+!!! must  
+
+Check hmi_capabilities, and respond to SDL providing the capabilities of supported buttons.  
```

Comment:
Could we please add a note here that if the HMI wishes to support soft buttons it must include CUSTOM_BUTTON in these capabilities?